util/encounter_tools.py

```diff
@@ -166,7 +166,13 @@ def choose_fight_times(args, encounters):
 
 # Timeline test/translate functions
 def clean_tl_line(line):
-    return line.split("#")[0]
+    # If a timeline text entry has a "#" character, this function breaks
+    # unless we do this little dance.
+    if line[0] is "#" or not re.search(r'"', line):
```

Comment:
What about `      # some comment`? I'm not sure I understand about checking `line[0]` here.

Reply:
Hmm, that's fair. What should we return in the event that check fails? 99% of the time we shouldn't get here in either case anyway. Maybe just return `None` and have the line be skipped by whatever script is calling it?

Reply:
I think it would be worth having `clean_tl_line` behave properly even in cases that it's not being called for right now.

What about something like:
(1) if there's no quotes, then remove after `#`
(2) if there's a `#` before a `"`, then remove after #
(3) your current split dance to only care about # after quotes